allensdk/brain_observatory/ecephys/ecephys_session.py

```diff
@@ -251,7 +262,54 @@ def get_lfp(self, probe_id):
 
         '''
 
-        return self.api.get_lfp(probe_id)
+        if mask_invalid_intervals:
+            fail_tags = ["all_probes", "probeA", "probeB", "probeC", "probeD", "probeE", "probeF"]
```

Comment:
1. I think this will apply a bad lfp mask from e.g. probeA to all probes
2. could you make the check a bit more generic using the `.probes` attribute?

Reply:
@NileGraddis good catch. Now I have used `.probes` to create a custom `fail_tags` for each probe